create

```diff
@@ -135,7 +135,8 @@ EOF
 UUID=$swap_uuid   swap            swap    defaults        0       0
 EOF
 
-cat > $TMPDIR/etc/network/interfaces <<EOF
+# Only create an interfaces file if the `ifupdown` package /etc/network directory exists.
+[[ -d "${TMPDIR}/etc/network" ]] && cat > "${TMPDIR}/etc/network/interfaces" <<EOF
```

Comment:
Thanks for your discovery and fix. Would you mind to use POSIX style conditional `[ ]` instead of bashism `[[ ]]`? It would match style for the reset of the script, even it it has bash in the shebang.

Reply:
Sure, can do that.